scripts/lib/CIME/SystemTests/system_tests_common.py

```diff
@@ -141,12 +141,12 @@ def run(self, skip_pnl=False):
             with self._test_status:
                 self._test_status.set_status(RUN_PHASE, TEST_PEND_STATUS)
 
+            resub_val = self._case1.get_value("RESUBMIT")
```

Comment:
I think this is close, but you cannot rely on case1/case2 existing for all test types.

Reply:
I think this is what you want

```
if (isinstance(self, SystemTestsCompareTwo)):
    resub_val = self._case1.get_value("RESUBMIT")
else:
    resub_val = 0
```